test/libsolidity/SyntaxTestParser.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "SyntaxTestParser.h"
+#include <fstream>
+#include <stdexcept>
+#include <boost/throw_exception.hpp>
+
+namespace dev
+{
+namespace solidity
+{
+namespace test
+{
+
+SyntaxTest SyntaxTestParser::parse(std::string const& _filename)
```

Comment:
We usually use `using namespace std` in cpp files to make it possible to faster grasp what is happening, but it is not required.